sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient_private.cs

```diff
@@ -87,12 +87,46 @@ private Request CreateRequest(HttpPipelineMethod method, params string[] path)
             return request;
         }
 
+        private Request CreateRequest(HttpPipelineMethod method, Uri uri)
+        {
+            Request request = _pipeline.CreateRequest();
+
+            request.Headers.Add(HttpHeader.Common.JsonContentType);
+            request.Headers.Add(HttpHeader.Names.Accept, "application/json");
```

Comment:
nit:  It seems that there's a content type header constant for JSON.   Is there maybe a corresponding one that we could use to eliminate the hardcoded `"application/json"`?

Reply:
@pakrym would it be ok fro me to add 
`public static readonly HttpHeader JsonAccept = new HttpHeader(Names.Accept, s_applicationJson);` into https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Pipeline/HttpHeader.cs#L60 ?

Reply:
I added it :|